hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -1233,9 +1233,7 @@ public void moveServers(Set<Address> servers, String targetGroupName) throws IOE
       Address firstServer = servers.iterator().next();
       RSGroupInfo srcGrp = getRSGroupOfServer(firstServer);
       if (srcGrp == null) {
-        // Be careful. This exception message is tested for in TestRSGroupsBase...
```

Comment:
TestRSGroupBase has been split to multiple classes since HBASE-21265 and this exception message is not used.